classes/Mail.php

```diff
@@ -109,6 +109,10 @@ public static function Send(
             $idShop = Context::getContext()->shop->id;
         }
 
+        if (is_numeric($idShop) && $idShop) {
+            $shop = new Shop((int) $idShop);
+        }
+
```

Comment:
Probably silently fail but this is the current behavior, I just made code a little bit _"clearer"_. I'd rather not change the behavior.